projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -30,6 +30,58 @@ <h2 class="row no-gutters section-header-heading">
               }}
             </div>
           </h2>
+          <ng-container *ngIf="oboCustomers$ | async as oboCustomers">
+            <ng-container *ngIf="checkActiveOboCustomers(oboCustomers.entries)">
+              <cx-fs-accordion-item
+                [opened]="true"
+                [title]="'fscommon.oboCustomers' | cxTranslate"
+              >
+                <div class="container">
+                  <div class="row justify-content-center">
+                    <ul class="col-12 accordion-list">
+                      <ng-container
+                        *ngFor="
+                          let oboCustomer of oboCustomers.entries;
+                          let i = index
+                        "
+                      >
+                        <ng-container *ngIf="oboCustomer.active">
+                          <li class="accordion-list-item row">
+                            <div class="col-6 col-sm-8">
+                              <div class="row no-gutters">
+                                <span class="col-12 col-sm-6 item-label">
+                                  {{ oboCustomer.name }}
+                                </span>
+                                <span class="col-12 col-sm-6 notice">
+                                  {{ oboCustomer.uid }}
+                                </span>
+                              </div>
+                            </div>
+                            <span
+                              [ngClass]="{
+                                'is-disabled':
+                                  (isCartTransferAllowedForSeller$ | async) &&
```

Comment:
In this `span` element you called `isCartTransferAllowedForSeller$ | async` twice. Maybe you can put it in `ng-container` wrapper element around the span?

Reply:
I thought about that, but I couldn't think of a solution to avoid not displaying any of those elements, because this condition is changing constantly, by clicking any of these elements. I am open to any suggestion.